doc/dev/deduplication.rst

```diff
@@ -157,25 +157,46 @@ How to use deduplication
 Ceph provides deduplication using RADOS machinery.
 Below we explain how to perform deduplication. 
 
+Prerequisite
+------------
+
+If the Ceph cluster is started from Ceph mainline, users need to check
+``ceph-test`` package which is including ceph-dedup-tool is installed.
```

Comment:
@jdurgin: Does this mean to say that the ``ceph-test`` package contains the ``ceph-dedup-tool``? What does "If the Ceph cluster is started from Ceph mainline" have to do with this?